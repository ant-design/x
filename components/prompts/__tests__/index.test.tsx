import React from 'react';

import Prompts from '..';
import mountTest from '../../../tests/shared/mountTest';
import rtlTest from '../../../tests/shared/rtlTest';

describe('bubble', () => {
  mountTest(() => <Prompts />);
  rtlTest(() => <Prompts />);
  beforeAll(() => {
    jest.useFakeTimers();
  });

  afterAll(() => {
    jest.useRealTimers();
  });

});
