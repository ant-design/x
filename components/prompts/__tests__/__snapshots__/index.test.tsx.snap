// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bubble rtl render component should be rendered correctly in RTL direction 1`] = `
<div
  class="ant-prompts ant-prompts-rtl"
>
  <ul
    class="ant-prompts-list"
  />
</div>
`;
